src/go/rdctl/cmd/listSettings.go

```diff
@@ -40,10 +64,92 @@ var listSettingsCmd = &cobra.Command{
 	},
 }
 
+// the reg file format is directly usable only on Windows,
+// but it can be created on any platform for purposes of testing or development
+
 func init() {
 	rootCmd.AddCommand(listSettingsCmd)
+	listSettingsCmd.Flags().StringVarP(&outputSettings, "output", "", "", fmt.Sprintf("output format: %s|%s[,hive][,type], default %s", jsonFormat, regFormat, jsonFormat))
```

Comment:
See https://github.com/rancher-sandbox/rancher-desktop/issues/4562

Reply:
I'm not sure what you mean by this

Reply:
Unless you mean it needs to be redone to support separate `--reg-hive` and `--section` options. That's done here now, and I'll address case-insensitivity where you bring it up.